### PR TITLE
Record clusters with data

### LINKED ISSUE #121: Support recordClustersWithData on Mastering projects
# 🙋 feature request

Support the `recordClustersWithData` dataset alias on Mastering projects

## 🤔 Expected Behavior

```
dataset = project.data_with_clusters()
```

## 💁 Possible Solution

No such method

## 🔦 Context

I want to publish data with clusters

## 💻 Examples

See above

### PR DESCRIPTION
**Pull request by Nelson-Andrade**
# ↪️ Pull Request

Fixes #121 

## ✔️ PR Todo

- [x] Added/updated testing for this change
- [x] Included links to related issues/PRs
- [x] Update relevant [docs](https://github.com/Datatamer/unify-client-python/tree/master/docs) + docstrings
- [x] Update the [CHANGELOG](https://github.com/Datatamer/unify-client-python/blob/master/CHANGELOG.md) under the current `-dev` version:
  - Add changelog entries under any that apply: **BREAKING CHANGES**, **NEW FEATURES**, **BUG FIXES**.
  - Changelog entry format: `[#<issue number>](<link to issue>) <change description>`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,8 +11,8 @@
   - [#86](https://github.com/Datatamer/unify-client-python/issues/86) Add support for creating projects
   - [#114](https://github.com/Datatamer/unify-client-python/issues/114) Add support for generating pairs estimate
   - [#106](https://github.com/Datatamer/unify-client-python/issues/106) Add support for initializing a source dataset
-  - [#107](https://github.com/Datatamer/unify-client-python/issues/107) Add support for creating a dataset attribute
```
Comment (nbateshaus) [inline CHANGELOG.md]: Looks like a merge conflict; bring #107 back!

File: tests/mock_api/test_continuous_mastering.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 import responses
 
+import responses
```
Comment (nbateshaus) [inline tests/mock_api/test_continuous_mastering.py]: Extraneous change; remove this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,7 +12,8 @@
   - [#114](https://github.com/Datatamer/unify-client-python/issues/114) Add support for generating pairs estimate
   - [#106](https://github.com/Datatamer/unify-client-python/issues/106) Add support for initializing a source dataset
   - [#107](https://github.com/Datatamer/unify-client-python/issues/107) Add support for creating a dataset attribute
-  
+  - [#121](https://github.com/Datatamer/unify-client-python/issues/121) Fetches record clusters with data represented as a dataset.
```
Comment (pcattori) [inline CHANGELOG.md]: Please update your `master` branch and put this under `0.6.0-dev`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,9 @@
 ## 0.6.0-dev
+  - [#121](https://github.com/Datatamer/unify-client-python/issues/121) Fetches record clusters with data represented as a dataset.
+  
   **BUG FIXES**
   - [#140](https://github.com/Datatamer/unify-client-python/issues/140) Dataset `itergeofeatures` now supports data with geo attribute NULL
-
+
```
Comment (pcattori) [inline CHANGELOG.md]: Remove whitespace:

```suggestion

```

**Pull request merged**